public/docs/_examples/structural-directives/dart/lib/heavy_loader_component.dart

```diff
@@ -0,0 +1,31 @@
+// #docregion
+import 'package:angular2/angular2.dart';
+import 'dart:async';
+
+int nextId = 1;
+
+@Component(
+    selector: 'heavy-loader',
+    template: '<span>heavy loader #{{id}} on duty!</span>')
+class HeavyLoaderComponent implements OnInit, OnDestroy {
+  int id = nextId++;
+  @Input() List<String> logs;
+
+  // Mock todo: get 10,000 rows of data from the server
+  ngOnInit() => _log(
+      "heavy-loader ${id} initialized, loading 10,000 rows of data from the server");
```

Comment:
@zoechi Aparently they're only necessary when it is a expression or you're calling a method in a object. I'll remove them. Thanks